src/index.less

```diff
@@ -194,6 +213,37 @@ ovh-chatbot { /* stylelint-disable-line selector-type-no-unknown */
           display: inline-block;
           position: relative;
           padding: 12px 14px;
+
+          a {
+            word-break: break-word;
+          }
+        }
+
+        .chatbot-toplist {
+          ol {
+            list-style: none;
+            margin: 0;
+            padding: 0;
+
+            li {
+              display: flex;
+              align-items: center;
+              margin-bottom: 10px;
+            }
+
+            li::before {
+              content: '';
+              background-image: url('data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iVVRGLTgiIHN0YW5kYWxvbmU9Im5vIj8+PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHZpZXdCb3g9IjAgMCA2NCA2NCI+PGNpcmNsZSBzdHlsZT0iZmlsbDpub25lO3N0cm9rZTojZmY2NDZjO3N0cm9rZS13aWR0aDozOyIgY3g9IjMyIiBjeT0iMzIiIHI9IjMwLjUiIC8+PHBhdGggc3R5bGU9ImZpbGw6bm9uZTtzdHJva2U6IzAwYTJiZjtzdHJva2Utd2lkdGg6NDtzdHJva2UtbGluZWNhcDpyb3VuZDtzdHJva2UtbGluZWpvaW46cm91bmQ7IiBkPSJtIDI4LjEzMDY0LDI0LjM3MjIyMSA3Ljc1ODU0LDcuNzU4NTQgLTcuNDk3MDEsNy40OTcwMSIvPjwvc3ZnPgo=');
```

Comment:
Is there any reason to not importing the svg file from `src/index.js` instead?

Reply:
If it's possible to import from less files, well yes, let's do that. I just didn't know.